tests/unit/services/key-manager-test.js

```diff
@@ -1,10 +1,9 @@
-import { getOwner } from '@ember/application';
```

Comment:
Another example of git diff winning.. call out any additional tests or changes that you made?

Reply:
Just cleaning up - no major changes